build.gradle

```diff
@@ -31,19 +31,24 @@ allprojects  {
   sourceCompatibility = 1.8
   targetCompatibility = 1.8
 
-  def idamBomVersion = '1.9.7'
+  def idamBomVersion = '2.0.1'
```

Comment:
can we not make most of these changes in the bom directly?

Reply:
If we control this repo, then maybe. But I'd rather do it properly, it seems that it's used in all web-apps, that would require a separate issue IMO so all affected apps can be tested.

Reply:
idam bom is ours and fixing it there will allow just bumping the bom in the other apps to fix this

Reply:
Done.